test/tests/functional/pbs_test_job_requeue_delay.py

```diff
@@ -0,0 +1,94 @@
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under a
+# commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestJobRequeueDelayAttribute(TestFunctional):
+    """
+    This test suite is for testing the new job_requeue_delay attribute
+    that replaced job_requeue_timeout.
+    """
+
+    def setUp(self):
+        PBSTestSuite.setUp(self)
+
+        if len(self.moms) != 2:
+            self.logger.error('test requires two MoMs as input, ' +
+                              '  use -p moms=<mom1>:<mom2>')
+            self.assertEqual(len(self.moms), 2)
+
+        # PBSTestSuite returns the moms passed in as parameters as dictionary
+        # of hostname and MoM object
+        self.momA = self.moms.values()[0]
+        self.momB = self.moms.values()[1]
+        self.momA.delete_vnode_defs()
+        self.momB.delete_vnode_defs()
+
+        self.hostA = self.momA.shortname
+        self.hostB = self.momB.shortname
+
+        rc = self.server.manager(MGR_CMD_DELETE, NODE, None, "")
+        self.assertEqual(rc, 0)
+
+        rc = self.server.manager(MGR_CMD_CREATE, NODE, id=self.hostB)
+        self.assertEqual(rc, 0)
+
+        rc = self.server.manager(MGR_CMD_SET, SERVER, {
+                                 'job_requeue_delay': 2}, expect=True)
+        self.assertTrue(rc)
+
+    def test_error_message(self):
+        j = Job(TEST_USER, attrs={ATTR_N: 'job_requeue_delay'})
+
+        test = []
+        test += ['dd if=/dev/zero of=file bs=1024 count=0 seek=1024\n']
```

Comment:
I have written this test case like I have written for PP-367 as seen in this [PR](https://github.com/PBSPro/pbspro/pull/135/files). The script works and the output file has the expected size. Demonstration below - 

[root@c7-2 tmp]# cat job.sh
dd if=/dev/zero of=file bs=1024 count=0 seek=30720
cat file
sleep 30

[root@c7-2 tmp]# qsub -lselect=vnode=c7 job.sh
6.c7-2
[root@c7-2 tmp]# qstat -s

c7-2: 
                                                            Req'd  Req'd   Elap
Job ID          Username Queue    Jobname    SessID NDS TSK Memory Time  S Time
--------------- -------- -------- ---------- ------ --- --- ------ ----- - -----
6.c7-2          root     workq    job.sh      77785   1   1    --    --  R 00:00
   Job run at Thu Nov 03 at 13:07 on (c7:ncpus=1)
[root@c7-2 tmp]# qrerun 6
qrerun: rerunning job is still in progress for 6.c7-2
[root@c7-2 tmp]# ls -l /var/spool/pbs/spool/
total 32768
-rw-------. 1 root root 24952832 Nov  3 13:07 6.c7-2.OU
[root@c7-2 tmp]# ls -lrth /var/spool/pbs/spool/
total 33M
-rw-------. 1 root root 30M Nov  3 13:07 6.c7-2.OU
-rw-------. 1 root root  76 Nov  3 13:07 6.c7-2.ER
[root@c7-2 tmp]#